core-services/prow/02_config/_plugins.yaml

```diff
@@ -986,6 +986,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/openshift/aws-load-balancer-controller/openshift-aws-load-balancer-controller-openshift-rel-1.*.yaml:
```

Comment:
what is this turning on?

Reply:
This is [the CI operator's technical detail](https://docs.ci.openshift.org/docs/how-tos/contributing-openshift-release/#ci-operator-configuration-sharding) which I didn't want to take time to understand :). It seems that all the operator configs are sharded into different configmaps depending on the branch naming. There are some default patterns like [this one for release-1.x branches](https://github.com/openshift/release/blob/d8cd82de0dfaede93759f75928478a140496e40c/core-services/prow/02_config/_plugins.yaml#L557-L562). This exact pattern [didn't allow me](https://redhat-internal.slack.com/archives/CBN38N3MW/p1683299021899709) to use `openshift-release-1.0` as the branch name. 
However no pattern matches `openshift-rel-1.0` branch name, so I had to add this explicit glob pattern to avoid the following error from `correctly-sharded-config` presubmit job:
```
time="2023-05-24T08:41:18Z" level=error msg="Validation failed" error="config_updater.maps: Invalid value: \"\": Config file does not belong to any auto-updating config: ci-operator/config/openshift/aws-load-balancer-controller/openshift-aws-load-balancer-controller-openshift-rel-1.0.yaml"
time="2023-05-24T08:41:18Z" level=error msg="Validation failed" error="config_updater.maps: Invalid value: \"\": Config file does not belong to any auto-updating config: ci-operator/jobs/openshift/aws-load-balancer-controller/openshift-aws-load-balancer-controller-openshift-rel-1.0-presubmits.yaml"
time="2023-05-24T08:41:18Z" level=error msg="Validation failed" error="config_updater.maps: Invalid value: \"\": Config file does not belong to any auto-updating config: ci-operator/jobs/openshift/aws-load-balancer-controller/openshift-aws-load-balancer-controller-openshift-rel-1.0-postsubmits.yaml"
time="2023-05-24T08:41:18Z" level=fatal msg="Found configurations that do not belong to the correct auto-updating config"
```

Reply:
ah got it thanks